lib/content-patcher/interface-patch-handler.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace NewspackContentConverter\ContentPatcher;
+
+interface PatchHandlerInterface {
```

Comment:
I am thinking about this one, and believe that we could remove it. I'll gladly remove it, but would rather wait a bit: to first write very simple tests for the PatchHandler, and the Plugin class which uses it, as well as to write some more patchers -- so that we don't remove it now, and need to add it later. But at this moment, I believe we'll be able to discard it.